### DISCUSSION TITLE
WebClient mouse cursor disappears while hovering

### DISCUSSION THREAD

**Discussion by calween** [rustdesk/rustdesk#5722] [Q&A]
Hello, 

I've tried recently run RustDesk WebClient using docker, but apparently it seems that mouse cursor disappears when you want to remotely control your device.
Therefore you cannot see what are you clicking at.
 
Did someone face this issue? Is there any solution for that?

Thanks in advance.

**Comment by rustdesk** [comment]
This version of webclient is not maintained yet (in beta > 1 year), we need find time to rewrite it. But we have no resource for the time being.

**Reply by calween** [reply]
Thanks for answering the question.
Anyway is there some rough estimation when it can be fixed?
